`gpctl completion bash` is failing.

### Bug description

On our private repository, we use `gpctl completion bash` in our prebuild tasks so we can easily manage our clusters when opening a workspace.

Recently, this command started to fail with the message:
```
panic: unable to redefine 't' shorthand in "build-batch" flagset: it's already used for "time-between-builds" flag

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc00021d100, 0xc000430320)
        /workspace/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:874 +0x6f0
github.com/spf13/pflag.(*FlagSet).AddFlagSet.func1(0xc000430320)
        /workspace/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:887 +0xa8
github.com/spf13/pflag.(*FlagSet).VisitAll(0xc000584f00, 0xc000521990)
        /workspace/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:290 +0x94
github.com/spf13/pflag.(*FlagSet).AddFlagSet(0xc00021d100, 0xc000584f00)
        /workspace/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:885 +0x58
github.com/spf13/cobra.(*Command).mergePersistentFlags(0x2673280)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:1644 +0x92
github.com/spf13/cobra.(*Command).LocalFlags(0x2673280, 0xc000584e00)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:1463 +0x2f
github.com/spf13/cobra.(*Command).LocalNonPersistentFlags(0x2673280, 0xc0003dd4d0)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:1453 +0xc5
github.com/spf13/cobra.writeFlags(0x1bc0e80, 0xc0003dd4d0, 0x2673280)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/bash_completions.go:534 +0x76
github.com/spf13/cobra.gen(0x1bc0e80, 0xc0003dd4d0, 0x2673280)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/bash_completions.go:647 +0x3fb
github.com/spf13/cobra.gen(0x1bc0e80, 0xc0003dd4d0, 0x2675300)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/bash_completions.go:629 +0xdf
github.com/spf13/cobra.gen(0x1bc0e80, 0xc0003dd4d0, 0x2675580)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/bash_completions.go:629 +0xdf
github.com/spf13/cobra.(*Command).GenBashCompletion(0x2675580, 0x1bc4ae0, 0xc000138008, 0xc000447d28, 0x0)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/bash_completions.go:661 +0xb2
github.com/gitpod-io/gitpod/gpctl/cmd.glob..func11(0x2673500, 0xc0003dadc0, 0x1, 0x1)
        /workspace/gitpod/gitpod-core/dev/gpctl/cmd/completion.go:61 +0x12c
github.com/spf13/cobra.(*Command).execute(0x2673500, 0xc0003dada0, 0x1, 0x1, 0x2673500, 0xc0003dada0)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x2675580, 0xc000000180, 0x200000003, 0xc000000180)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /workspace/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/gitpod-io/gitpod/gpctl/cmd.Execute()
        /workspace/gitpod/gitpod-core/dev/gpctl/cmd/root.go:31 +0x31
main.main()
        /workspace/gitpod/gitpod-core/dev/gpctl/main.go:14 +0x25
```

### Steps to reproduce

1. Install gpctl
2. run `source <(gpctl completion bash)`

### Expected behavior

Completion to be installed with no failures

### Example repository

https://github.com/gitpod-com/gitpod

### Anything else?

_No response_

--- a/dev/gpctl/cmd/imagebuilds-build-batch.go
+++ b/dev/gpctl/cmd/imagebuilds-build-batch.go
@@ -111,5 +111,5 @@ func init() {
 	imagebuildsCmd.AddCommand(imagebuildsBuildBatch)
 
 	imagebuildsBuildBatch.Flags().Bool("force-rebuild", false, "force an image build even if the image exists already")
-	imagebuildsBuildBatch.Flags().IntP("time-between-builds", "t", 0, "wait N seconds between starting builds")
+	imagebuildsBuildBatch.Flags().IntP("time-between-builds", "", 0, "wait N seconds between starting builds")
 }

